m4_sessions/1710-inning/group_two.md

```diff
@@ -1,10 +1,30 @@
 Group Two
 
-Session Title: 
+Session Title: Rule #6 - Don't take yourself so seriously...
+
 Description: Why should students come to this session? 
+Learn why it's important to take a step back. Get a better perspective and a more light hearted approach to...everything. 
+
+You're not that important. Laugh at the joke called "you"?
+
+How does your mentality affect growth, perception and life in general.
+
+What's really important?
+
 
 Session Outcomes: What will students leave with or gain from this session?
 
+Perspective. Identifying where and how to let go. What's the balance between serious and too serious? (and candy).
+
 Session Outline:  Include the flow of the session and the time for each section.
 
+1 hour 15 minutes total
+10 minutes - introduction
```

Comment:
Be sure to build in an ice breaker or introductions for students. They will be a mix of mods and programs